hack/update-generated-defaulters.sh

```diff
@@ -5,18 +5,19 @@ os::build::setup_env
 
 os::util::ensure::built_binary_exists 'gendefaults'
 
-PREFIX=github.com/openshift/origin
+PREFIX=github.com/openshift/origin/
 
 INPUT_DIRS=$(
-  find . -not  \( \( -wholename '*/vendor/*' \) -prune \) -name '*.go' | \
-	xargs grep --color=never -l '+k8s:defaulter-gen=' | \
-	xargs -n1 dirname | \
-	sed "s,^\.,${PREFIX}," | \
-	sort -u | \
-	paste -sd,
+  cd ${OS_ROOT}
```

Comment:
The contract from `hack/lib/init.sh` is that you are in `$OS_ROOT` -- no need for `cd`